.github/scripts/close-invalid-link.cjs

```diff
@@ -35,11 +35,11 @@ module.exports = async ({ github, context }) => {
 
     try {
         const link = issue.data.body.match(/(https?:\/\/github.com\/.*)/)[0];
-        const isValidLink = (await fetch(link)).status === 200;
+        const isValidLink = (await fetch(link)).statusText === 'OK');
```

Comment:
Actually we’d want the `.ok` property of the response; the status text ‘OK’ is only for 200

Reply:
Thank you! TIL